libs/env/index.js

```diff
@@ -1,12 +1,15 @@
 import fs from "node:fs";
 import path from "node:path";
-import { cwd } from "node:process";
+import { fileURLToPath } from "node:url";
 
 import dotenv from "dotenv";
 
 import { VALID_FLAW_CHECKS } from "../constants/index.js";
 
-export const ROOT = path.join(cwd(), ".");
+// Spread into two lines to get the ROOT path,
+// to prevent WebPack from treating 'new URL("../..", import.meta.url)' as an import.
```

Comment:
Small capitalization nit, LGTM otherwise!

```suggestion
// to prevent Webpack from treating 'new URL("../..", import.meta.url)' as an import.
```